src/constants.ts

```diff
@@ -18,7 +18,7 @@ export const DEFAULT_BRANCH_PATTERNS: RegExp[] = [/^master$/, /^production$/, /^
 /**
  * Regex to match JIRA issue keys.
  */
-export const JIRA_REGEX_MATCHER = /\d+-(([A-Z0-9]{1,10})|[a-z0-9]{1,10})/g;
+export const JIRA_REGEX_MATCHER = /(GNA-[0-9]{3,})/g;
```

Comment:
why no prt on this?
also I recommend starting with ^ and removing the g as we only need 1 match right?

Reply:
Yep thanks, for the 'tag fix version' regex I explicitly excluded PRT since those don't get tagged, but you're probably right, for the branch checks we should allow PRT Jiras. Have updated.